<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimepointChooser</class>
 <widget class="QDialog" name="TimepointChooser">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Timepoints</string>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_FromData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Get From Data</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="button_FromData">
            <property name="text">
             <string>Get Timepoints from Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>or</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Calculate">
         <property name="title">
          <string>Calculate</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_StartTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Start Time</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_StartTime"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_EndTime">
              <property name="text">
               <string>End Time</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_EndTime"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_NumIntervals">
              <property name="text">
               <string># Intervals</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_NumIntervals"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_IntervalSize">
              <property name="text">
               <string>Interval Size</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_IntervalSize"/>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="button_Calculate">
              <property name="text">
               <string>Calculate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Timepoints">
       <property name="title">
        <string>Timepoints (space-separated)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Timepoints">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>368</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelTimepointsInvalid">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>The list of time points is invalid! (It may only contain digits, e.g. &quot;12.5&quot;, delimited by white space.)</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TimepointChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TimepointChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
